pyproject.toml

```diff
@@ -6,7 +6,7 @@ name = "aiokafka"
 description = "Kafka integration with asyncio"
 readme = "README.rst"
 requires-python = ">=3.9"
-license = { file = "LICENSE" }
+license = { text = "Apache-2.0" }
```

Comment:
According to PEP, this ways is [deprecated](https://peps.python.org/pep-0639/#deprecate-license-key-table-subkeys):

> Table values for the `license` key in the `[project]` table, including the `text` and `file` table subkeys, are now deprecated.

```suggestion
license = "Apache-2.0"
```

Reply:
Yeah. Unfortunately setuptools doesn't support the new syntax yet. Using `license.text` until then will make updating later a bit easier and also help with readability on PyPI.

**before**

<img width="278" alt="Screenshot 2024-11-19 at 08 37 26" src="https://github.com/user-attachments/assets/08e2834f-9820-41a8-80a0-b15a09e1d605">

**after**

<img width="248" alt="Screenshot 2024-11-19 at 08 38 02" src="https://github.com/user-attachments/assets/fdb2236c-497a-4295-9684-0512f8f0a82b">

Reply:
Probably it's time to switch to `hatch` (need to figure out how to deal with `cython` to do this). If not, I'd rather wait until `setuptools` is fixed.